packages/components/src/Popover/Popover.module.scss

```diff
@@ -127,7 +127,7 @@ $cautionary-box-border-color: var(--color-yellow-700);
   display: flex;
   align-items: center;
   white-space: nowrap;
-  padding-right: $spacing-md;
+  padding-inline-end: $spacing-md;
```

Comment:
Nice, could potentially update this to spacing-24, or even the css var if you're keen :)